lib/attrsets.nix

```diff
@@ -171,15 +187,12 @@ rec {
      Example:
        filterAttrsRecursive (n: v: v != null) { foo = { bar = null; }; }
        => { foo = {}; }
+
+     Type:
+       filterAttrsRecursive :: (String -> Any -> Bool) -> AttrSet -> AttrSet
   */
   filterAttrsRecursive =
-    /*
-String -> Any -> Bool
```

Comment:
type information is lost when this part of the comment is removed as of now.

Reply:
I'm leaving that out on purpose because the formatting is bad. The type is still captured by the overall type signature.